pkgs/development/libraries/libvirt/default.nix

```diff
@@ -93,6 +93,8 @@ in stdenv.mkDerivation rec {
     "sysconfdir=$(out)/var/lib"
   ];
 
+  binPath = [ iptables iproute pmutils numad numactl bridge-utils dmidecode dnsmasq ebtables ] ++ optionals enableIscsi [ openiscsi ];
```

Comment:
Although this works, it might not do exactly what you want, as ```binPath``` leaks as the environment variable.
For only declaring a local value it should be
```
postInstall = let
  binPath = [ ... ];
in ''
```

Reply:
Like this?

Reply:
Yes, it would be nice to fix.
At least to reduce spreading of bad examples